plopfile.js

```diff
@@ -88,4 +88,31 @@ module.exports = function (plop) {
       },
     ],
   });
+  plop.setGenerator("create-docs", {
```

Comment:
So we can extend for patterns etc
```suggestion
  plop.setGenerator("create-component-docs", {
```